### PR TITLE
DM XMLs 1.4 tag

### PR DESCRIPTION
**Pull request by cecille**
cherry-pick of https://github.com/project-chip/connectedhomeip/pull/36153 onto 1.4 branch

### CODE REVIEW

**Review by robszewczyk — APPROVED** [review]

**Review by cjandhyala — APPROVED** [review]

**Review by jamesharrow — CHANGES_REQUESTED** [review]
EVSE PREF should have had P removed (see new spec PR https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10597)

**Review by jamesharrow — COMMENTED** [review]
Removing request changes

**Review by jamesharrow — APPROVED** [review]

File: data_model/1.4/clusters/EnergyEVSE.xml [hunk 1]
```diff
@@ -240,7 +240,6 @@ Davis, CA 95616, USA
     <struct name="ChargingTargetScheduleStruct">
```
Comment (jamesharrow) [inline data_model/1.4/clusters/EnergyEVSE.xml:240]: EVSE PREF should not be P

Comment (cecille) [reply data_model/1.4/clusters/EnergyEVSE.xml:237]: this has been resolved now with the latest spec scrape.

**Review by woody-apple — APPROVED** [review]

**Review by woody-apple — APPROVED** [review]

**Pull request merged**